components/atoms/Text.js

```diff
@@ -1,30 +1,8 @@
-import styled from 'styled-components'
+import React from 'react'
+import { Text as RebassText } from 'rebass/styled-components'
 
-import {
-  space,
-  fontSize,
-  fontWeight,
-  textStyle,
-  textAlign,
-  color,
-  width
-} from '../util'
+const Text = props => (
```

Comment:
Why don't we just import this and do the override inside of the main component index?

Reply:
I was anticipating to come back and add some customization after running tests or review. So I left the file here. We can just directly export it like we do for `Flex` and `Box`. It is easier to maintain any overrides in its own file.